Vizzuality/half-earth-v3
# Fix/nrc tooltip

## Fix NRC tooltip
### Description
NRC tooltip was not working as the search logic was using a different action than the tooltip expected. Now we share the actions and selectors for all the map tooltips. Also some renames were made to clarify

### Testing instructions
Go to the NRC landing page and search for a country. The tooltip should appear after search. Also the AOI tooltip should be working
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-182?atlOrigin=eyJpIjoiNTg3MjhiNDFlY2I2NDI0MmFhZjk0ZjJlNDZhYjY5OWUiLCJwIjoiaiJ9
